dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
       {
         "value": "564GE",
-        "label": "JAY CBOC"
+        "label": "Jay VA Clinic"
       },
       {
         "value": "623GB",
-        "label": "VINITA CBOC"
+        "label": "Vinita VA Clinic"
       },
       {
         "value": "635GE",
-        "label": "STILLWATER CBOC"
+        "label": "Stillwater VA Clinic"
       },
       {
         "value": "635GF",
-        "label": "ALTUS CBOC"
+        "label": "Altus VA Clinic"
       },
       {
         "value": "635GG",
-        "label": "ENID CBOC"
+        "label": "Enid VA Clinic"
       },
       {
         "value": "635QB",
-        "label": "SOUTH OKLAHOMA CITY VA CLINIC"
+        "label": "South Oklahoma City VA Clinic"
       },
       {
         "value": "623QC",
-        "label": "YALE AVENUE VA CLINIC"
+        "label": "Yale Avenue VA Clinic"
       },
       {
         "value": "635QC",
-        "label": "FOURTEENTH STREET VA CLINIC"
+        "label": "Fourteenth Street VA Clinic"
       },
       {
         "value": "623",
-        "label": "MUSKOGEE, OK VAMC"
+        "label": "Jack C. Montgomery Department of Veterans Affairs Medical Center"
       },
       {
         "value": "635",
-        "label": "OKLAHOMA CITY VA MEDICAL CENTER"
+        "label": "Oklahoma City VA Medical Center"
       }
     ],
     "NV": [
       {
         "value": "654GB",
-        "label": "CARSON VALLEY VA CLINIC"
+        "label": "Carson Valley VA Clinic"
       },
       {
         "value": "593GC",
-        "label": "PAHRUMP CBOC"
+        "label": "Pahrump VA Clinic"
       },
       {
         "value": "660GC",
         "label": "ELY VA CLINIC"
       },
       {
         "value": "654GC",
-        "label": "LAHONTAN VALLEY VA CLINIC"
+        "label": "Lahontan Valley VA Clinic"
       },
       {
         "value": "593GD",
-        "label": "NORTHWEST PCC"
+        "label": "Northwest Las Vegas VA Clinic"
       },
       {
         "value": "593GE",
-        "label": "SOUTHEAST PCC"
+        "label": "Southeast Las Vegas VA Clinic"
       },
       {
         "value": "593GF",
-        "label": "SOUTHWEST PCC"
+        "label": "Southwest Las Vegas VA Clinic"
       },
       {
         "value": "593GG",
-        "label": "NORTHEAST PCC"
+        "label": "Northeast Las Vegas VA Clinic"
       },
       {
         "value": "654GE",
-        "label": "RENO EAST VA CLINIC"
+        "label": "Reno East VA Clinic"
       },
       {
         "value": "593GH",
-        "label": "JESSE DEAN VA CLINIC"
+        "label": "Master Chief Petty Officer Jesse Dean VA Clinic"
       },
       {
         "value": "593QC",
-        "label": "WEST CHEYENNE VA CLINIC"
+        "label": "West Cheyenne VA Clinic"
       },
       {
         "value": "654QD",
-        "label": "VIRGINIA STREET VA CLINIC"
+        "label": "Virginia Street VA Clinic"
       },
       {
         "value": "593",
-        "label": "SOUTHERN NEVADA HCS"
+        "label": "North Las Vegas VA Medical Center"
```

Comment:
Let's confirm this name change.

Reply:
This is going from an entire Health Care System (with seven facilities) to a single facility. Since Facility Locator gives us this single facility for this code, I suppose we should just stick with what it provides.